taca/illumina/NovaSeq_Runs.py

```diff
@@ -1,30 +1,16 @@
-import os
-import re
-import csv
-import glob
-import shutil
-import gzip
-import operator
-import subprocess
-from datetime import datetime
-from taca.utils.filesystem import chdir, control_fastq_filename
-from taca.illumina.Runs import Run
 from taca.illumina.HiSeqX_Runs import HiSeqX_Run
-from taca.utils import misc
-from flowcell_parser.classes import RunParametersParser, SampleSheetParser, RunParser, LaneBarcodeParser, DemuxSummaryParser
-
 
 import logging
```

Comment:
Wouldn't this be imported from HiSeqX_Run as well?

Reply:
logging? Ah yeah that's right I think. Should be able to remove it from all the subclasses 👍